.github/ISSUE_TEMPLATE/weakness_report.yml

```diff
@@ -0,0 +1,92 @@
+name: Weakness Report
+description: Report a weakness in the sense of the CWE; a potential source of vulnerabilities.
+#title: ""
+labels: ["weakness"]
```

Comment:
```suggestion
labels: ["security"]
```
As Noon said, I think let's make this a general security issue report and add "weakness" ourselves.